rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    pesrson.rentals << self
+
+    @book = book
+    book.rentals << self
+  end
+end
```

Comment:
In order to be able to read as well as change the values of book and person in the rental class, you need to create the getters and setters for them, so kindly add the @book and @person to the attr_accessor instead of the attr_reader. Remember, the attr_reader method helps to read the value only but can not change or alter the value  :+1:

Reply:
Okay. thanks